sqlite.go

```diff
@@ -1298,12 +1298,12 @@ func (c *conn) close() (err error) {
 	return err
 }
 
-type sqlite struct {
+type SQLite struct {
```

Comment:
Please rename this type to Driver. I avoided that name as it collides with the qualifier of import sql/driver. The collision disappears when its an exported name. It will also look a much better in the type switch case, *sqlite.Driver instead of *sqlite.Sqlite.